src/System.Security.Cryptography.X509Certificates/tests/CertificateCreation/CertificateRequestChainTests.cs

```diff
@@ -179,6 +179,7 @@ public static void ChainCertRequirements(bool useIntermed, bool? isCA, X509KeyUs
                     chain.ChainPolicy.RevocationMode = X509RevocationMode.NoCheck;
                     chain.ChainPolicy.VerificationFlags = X509VerificationFlags.AllowUnknownCertificateAuthority;
                     chain.ChainPolicy.ExtraStore.Add(rootCert);
+                    chain.ChainPolicy.VerificationTime = start.ToLocalTime().DateTime;
```

Comment:
If this is in local time, what happens if we hit a daylight savings time transition just after we fetch the local time?